app/assets/javascripts/components/timeline/timeline.jsx

```diff
@@ -228,52 +228,53 @@ const Timeline = createReactClass({
               addWeek={this.props.addWeek}
             />
           </div>
-        )
-
-        );
-        i += 1;
-      } else {
-        const weekAnchorName = `week-${i + 1 + weeksBeforeTimeline}`;
-        weekNavInfo.push({ emptyWeek: false, title: week.title });
-        weekComponents.push((
-          <div key={week.id}>
-            <a className="timeline__anchor" name={weekAnchorName} />
-            <Week
-              week={week}
-              index={i + 1}
-              reorderable={this.props.reorderable}
-              editableTitles={this.props.editableTitles}
-              usingCustomTitles={usingCustomTitles}
-              updateTitle={this.props.updateTitle}
-              blocks={week.blocks}
-              deleteWeek={this.deleteWeek.bind(this, week.id)}
-              meetings={this.props.week_meetings[i]}
-              timeline_start={this.props.course.timeline_start}
-              timeline_end={this.props.course.timeline_end}
-              all_training_modules={this.props.all_training_modules}
-              editableBlockIds={this.props.editableBlockIds}
-              edit_permissions={this.props.edit_permissions}
-              saveBlockChanges={this.props.saveBlockChanges}
-              setBlockEditable={this.props.setBlockEditable}
-              cancelBlockEditable={this.props.cancelBlockEditable}
-              updateBlock={this.props.updateBlock}
-              addBlock={this.props.addBlock}
-              deleteBlock={this.props.deleteBlock}
-              saveGlobalChanges={this.props.saveGlobalChanges}
-              canBlockMoveUp={this._canBlockMoveUp.bind(this, week, weekIndex)}
-              canBlockMoveDown={this._canBlockMoveDown.bind(this, week, weekIndex)}
-              onMoveBlockUp={this._handleMoveBlock.bind(this, true)}
-              onMoveBlockDown={this._handleMoveBlock.bind(this, false)}
-              onBlockDrag={this._handleBlockDrag}
-              weeksBeforeTimeline={weeksBeforeTimeline}
-              trainingLibrarySlug={this.props.course.training_library_slug}
-              current_user={this.props.current_user}
-            />
-          </div>
-        )
-        );
+        )        );
+        j += 1;
```

Comment:
I kind of overlooked the previous suggestion. Now I did this.props.week_meetings[i] && this.props.week_meetings[i].length === 0, it looks nice, thanks. I hope it will work